infrastructure/modules/kms/main.tf

```diff
@@ -0,0 +1,50 @@
+resource "aws_kms_key" "encryption_key" {
+  description         = var.kms_key_description
+  policy              = data.aws_iam_policy_document.kms_key_policy_doc.json
+  enable_key_rotation = var.kms_key_rotation_enabled
+}
+
+resource "aws_kms_alias" "encryption_key_alias" {
+  name          = var.kms_key_name
+  target_key_id = aws_kms_key.encryption_key.id
+}
```

Comment:
tags

Reply:
added but then removed from the alias resource as tags not allowed on aliases